src/ProjectTemplates/test/BaselineTest.cs

```diff
@@ -16,7 +17,8 @@ public class BaselineTest
     {
         public BaselineTest(ProjectFactoryFixture projectFactory, ITestOutputHelper output)
         {
-            Project = projectFactory.CreateProject(output);
+            ProjectFactory = projectFactory;
+            Output = output;
         }
 
         public Project Project { get; set; }
```

Comment:
You may as well get rid of this it you're not generating the project in the constructor.